packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -321,6 +321,11 @@ type BaseInputCommonProps = FormInputLabelProps &
      * @default text
      **/
     valueComponentType?: 'text' | 'heading';
+    /**
+     * Whether to render the input as a table cell
+     * @default true
+     **/
+    isTableInputCell?: boolean;
```

Comment:
_Nit:_ 
I wonder if we should enhance base input to make it work/look like table cell input or should we just created a separate TableCellInput component with very minimal styles/jsx. Since most of the things from BaseInput are disabled in table cell input (eg: validation texts) since it has very different method of showing validation messages (eg inside tooltip). Just seeing lots of conditional rendering basis of IsTableInputCell.

This is fine for now, but we can reevaluate this later on to see if things can be better organized/refactored.

Reply:
Ideally all are inputs should come from a single place. This is the same on design. If tomorrow we need to enable a feature that's there on other inputs, it'll be difficult if we maintain a different input. But also the input is getting very complex now and we might be better off splitting it